### PR TITLE
feat(lib): Trim snyk url from utm_* query params on the results

### PR DESCRIPTION
**Pull request by zivkaziv**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

[#35 ](https://github.com/lirantal/is-website-vulnerable/issues/35)
## Motivation and Context
Removing utm_* query params of snyk analytics

## How Has This Been Tested?
Adding unit tests for different Urls with different scenarios of query params
Running is-website-vulnerable https://go.microsoft.com and check the snyk url is printed without utm_* query params
Running is-website-vulnerable https://go.microsoft.com --json and check the snyk url is printed without utm_* query params

## Checklist:

- [ ] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I added a picture of a cute animal cause it's fun

Alt Text

### CODE REVIEW

**Comment by zivkaziv** [comment]
Regading the way I'm triming the utm_*
I'm using regex which was detected as problematic by eslint. 
I've disabled it on this line, let me know if that's ok.. 

Thank you:)

File: src/Utils.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 const url = require('url')
+// Removing utm_* parameters from URL
+// eslint-disable-next-line security/detect-unsafe-regex
+const REGEX_UTM_TRIMMER = /(\?)utm[^&]*(?:&utm[^&]*)*&(?=(?!utm[^\s&=]*=)[^\s&=]+=)|\?utm[^&]*(?:&utm[^&]*)*$|&utm[^&]*/gi
```
Comment (lirantal) [inline src/Utils.js]: @zivkaziv any chance we can do it without a regex?
one idea is perhaps to transform it to a structured URL, and then re-build it again to a URL string but excluding those query params that you remove here.

Comment (zivkaziv) [reply src/Utils.js]: Sure.. I'll use your suggestion:)

Comment (lirantal) [reply src/Utils.js]: Thanks! 
http://nodejs.org/api/url.html should set you up well

**Pull request merged**